tasks/serializers.py

```diff
@@ -0,0 +1,56 @@
+from rest_framework import serializers
+from .models import Task
+
+
+class CreateTaskSerializer(serializers.ModelSerializer):
+
+    share_link = serializers.CharField(default=None)
+    share_text = serializers.CharField(default=None)
+    share_img = serializers.ImageField(default=None)
```

Comment:
```suggestion
```
This is redundant I think. correct me if I am wrong!

Reply:
If I not use these, I suppose those fields do not get the `None` value and so the following error pops up...
```
line 264, in test_success_authorised_member
    self.assertEqual(response.status_code, status.HTTP_201_CREATED)
AssertionError: 400 != 201
```

Reply:
Actually the share link you gave in valid payload is more than 100 characters. :laughing:

Reply:
Ahaa.. I get a Key error instead now without these, 🤣🤣
```
teesco-backend/tasks/serializers.py", line 38, in save
    if valid_data['share_text'] is None:
KeyError: 'share_text'

----------------------------------------------------------------------
Ran 13 tests in 3.938s

FAILED (errors=1)
```

Reply:
@naveennvrgup sir, Error has been resolved and these lines have been removed, thanks to @anshsrtv 🎉😄

Reply:
kudos @anshsrtv :clap: